tests.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+import zipfile
+import requests
+from shutil import copyfile
+copyfile('lichess-bot.py', 'lichess_bot.py')
```

Comment:
Alternatively, you could do this:

    import importlib
    lichess_bot = importlib.import_module("lichess-bot")

This way, there's no extra file left over after the tests.